hal/sai_api.h

```diff
@@ -174,14 +174,24 @@ sai_result_t sai_init(sai_t *obj, sai_init_t *init);
  * - on reception : will read a sample from the fifo to the nether.
  * - on transmission : will send a '0'.
  */
-bool sai_xfer(sai_t *obj, uint32_t *psample);
+bool sai_transfer(sai_t *obj, uint32_t *psample);
 
 /**
  * Releases & de-initialize the referenced peripherals.
  * @param obj       This SAI instance.
  */
 void sai_free(sai_t *obj);
 
+
+/**
+ * Checks init parameter sanity.
+ * @param init      A config to be sanity checked.
+ * @return  True if the init is sane.
+ * @note It does not guaranty that this configuration will be supported by the
```

Comment:
hm, apparently `guaranty` is an obsolete form of `guarantee` which has a close meaning to `warranty` but not `warrantee` from a legal point of vue.
I'll rephrase that to avoid any ambiguity.